### PR TITLE
Alignment metrics

### PR DESCRIPTION
**Pull request by jonchang**
# Pull Request

## Description
Initial WIP for metrics work regarding image alignment

## Related Issues
https://github.com/CDCgov/IDWA/issues/115

## Checklist
Please review and complete the following checklist before submitting your pull request:

- [x] I have ensured that the pull request is of a manageable size, allowing it to be reviewed within a single session.
- [x] I have reviewed my changes to ensure they are clear, concise, and well-documented.
- [x] I have updated the documentation, if applicable.
- [x] I have added or updated test cases to cover my changes, if applicable.
- [x] I have minimized the number of reviewers to include only those essential for the review.
- [x] I have notified teammates in the review thread to build awareness.

## Checklist for Reviewers
Please review and complete the following checklist during the review process:

- [ ] The code follows best practices and conventions.
- [ ] The changes implement the desired functionality or fix the reported issue.
- [ ] The tests cover the new changes and pass successfully.
- [ ] Any potential edge cases or error scenarios have been considered.

### CODE REVIEW

**Review by arinkulshi-skylight — DISMISSED** [review]
LGTM

File: OCR/tests/alignment_test.py [hunk 1]
```diff
@@ -15,13 +15,23 @@
...
     def test_alignment_filled(self):
         aligner = ImageHomography(template_image_path)
-        warped_image = np.array(RandomPerspectiveTransform(filled_image_path).transform(distortion_scale=0.1))
+        warped_image = np.array(RandomPerspectiveTransform(filled_image_path).random_transform(distortion_scale=0.1))
         aligned = aligner.transform_homography(warped_image)
         res = cv.absdiff(aligner.template, aligned)
         assert aligner.template.shape == warped_image.shape
         assert np.median(res) == 0
+
+    def test_inverse_alignment(self):
```
Comment (zdeveloper) [inline OCR/tests/alignment_test.py:29]: can you please add some comments for someone who is not super familiar with this code
- a high level description of this test, for example: we are warping an image with a known values, then we estimate the warp, then compare the estimated value to the original known value and asserting that the difference is not too much.
- why the value of 1 was selected
-

**Review by zdeveloper — APPROVED** [review]
LGTM! thank you!
